web/css/otherservices.css

```
@@ -0,0 +1,99 @@
+.otherservices{
+	width: 100%;
+	height: 750px;
+}
+.otherservicescontent{
+	margin-left: 100px;
+	margin-right: 100px;
+	display: flex;
+	justify-content: space-around;
+
+}
+.content-flex{
+	width: 350px;
+	height: 150px;
+	margin-top: 100px;
+
+}
+.content-p1{
+	margin-top: 0px;
+	margin-left: 60px;
+	font-size: 18px;
+	color: #494949;
+	font-weight: 600;
+	font-family: "Raleway";
+
+}
+.content-p2{
+	font-size: 14px;
+	line-height: 26px;
+	color: #7c7c7c;
+	font-weight: 500;
+	font-family: "Raleway";
+}
+.developing{
+	width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/ideadeveloping.png') left no-repeat;
+
+}
+.coding{
+	width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/bestcoding.png') left no-repeat;	
+
+}
+.gamedesign{
+	width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/gamedesign.png') left no-repeat;	
+
+}
+.responsivedesign{
+		width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/responsivedesign.png') left no-repeat;	
+
+}
+.freeshopping{
+		width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/freeshopping.png') left no-repeat;	
+
+}
+.bonusdownload{
+		width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/bonusdownload.png') left no-repeat;	
+
+}
```

ТАК Я НЕ ПОНЯЛ?!! Это что ещё закопипаста??? Выдерни это к чертовой матери!! Я бы вообще стёр это всё, и через имг сделал бы, но раз тебе нравится через css, то это вообще в пару строк можно.
Пиши:
```suggestion
.otherservicescontent ul {
	width: 330px;
	height: 50px;
	padding: 10px;
	padding-top: 5px;
	padding-left: 0px;
	margin: 0px;
	list-style-type: none;
}

.developing{ background: url('../img/3otherServices/ideadeveloping.png') left no-repeat; }
.coding{ background: url('../img/3otherServices/bestcoding.png') left no-repeat; }
.gamedesign{ background: url('../img/3otherServices/gamedesign.png') left no-repeat; }
.responsivedesign{ background: url('../img/3otherServices/responsivedesign.png') left no-repeat; }
.freeshopping { background: url('../img/3otherServices/freeshopping.png') left no-repeat; }
.bonusdownload { background: url('../img/3otherServices/bonusdownload.png') left no-repeat; }
```